### ISSUE TITLE
Aero Shake and Mica Material support on windows 

### ISSUE THREAD

**Issue by Frank-99** [JFormDesigner/FlatLaf#584]
Hi, I noticed that the windows Aero Shake isn't working, is this a known issue already?

Also, would it be possible to have the Mica material be supported by design in some future?
I tried to rework the code in the git linked below that i found. At first i managed to get it working quite nicely on its own, but then when i pared it with the FlatLaf library for having the benefit of a custom title bar and more control of the frame overall, I just wasn't able to make it work together...

Here is the repository:
https://github.com/mimoguz/custom_window

I know I shouldn't be too hopeful of the latter request, but it's an amazing library and it would be perfect if it managed to support native OS stiling of the window as an option!!

Thank you in advance!

**Comment by DevCharly** [comment]
Hmm, Aero Shake works for me, if enabled in Windows Settings. Tested on Windows 10 and 11.
Are you sure that it is enabled on your system?

**Comment by Frank-99** [comment]
Yes, it works with all my other apps but couldn't get it working with mine
🤔
Will try to check again more thoroughly though, just to be sure. But I'm
positive it wasn't working...

**Comment by Frank-99** [comment]
I can confirm that aero shake works, it started working after i updated windows 11 to the latest OS version...
It stopped working System-wide at some point and just noticed testing my java app

**Comment by DevCharly** [comment]
If I understand the [Mica material](https://learn.microsoft.com/en-us/windows/apps/design/style/mica) docs correctly, then Mica uses slightly different background colors for windows to create visual hierarchy. It also uses desktop wallpaper in some way.

Since Swing paints the window background itself, I don't think that it is possible to use Windows Mica material in Swing.
